.golangci.yaml

```
@@ -514,8 +513,4 @@ linters:
       # Type defining config.toml and hotkey.toml
       - path: 'src/internal/common/config_type.go'
         linters:
-          - lll 
-          # TODO: Fix this. There are "bad syntax for struct tag value" errors
-          - govet   
-          # TODO: Fix this. There are "SA5008: unparseable struct tag" errors
-          - staticcheck
\ No newline at end of file
+          - golines 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)